test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6033,6 +6033,13 @@ def submit(self, obj, script=None, extend=None, submit_dir=None):
                     del obj.custom_attrs[ATTR_queue]
 
             _conf = self.default_client_pbs_conf
+            user_host = PbsUser.get_user(obj.username).host
+            if (not self._is_local) or \
+                    (not self.du.is_localhost(user_host)):
+                for k, v in obj.custom_attrs.items():
+                    if isinstance(v, str) and (" " in v):
```

Comment:
what if v has `"` or `'`? don't we want to put under quotes that value also?

Reply:
@hirenvadalia I have tried with strings having 'or " in it("This'is'test" and "This"is"test") but it doesn't have any effect as in string is submitted as "Thisistest" which doesn;t have any single quotes.

Reply:
@shilpakodli Assuming you are talking about running qsub command from the shell, not PTL, I think you missed to escape those quotes while running qsub command in the shell.